### DISCUSSION TITLE
Some lines when boot before deepin logo

### DISCUSSION THREAD

**Discussion by ghost** [linuxdeepin/developer-center#3226] [Q&A | 问答板块]
Hi,
I have a question, is the line that appear on boot before deepin logo shows safe or indicates some kind of an issue?
![IMG20220606110317](https://user-images.githubusercontent.com/98664445/172122565-df0ccaad-5034-4536-8ef3-7c541c5224c7.jpg)

**Comment by BLumia** [comment]
You can simply Google that error to get more detail. It's a common error that is worth not worrying about too much if the machine is running fine :)

**Reply by ghost** [reply]
> You can simply Google that error to get more detail. It's a common error that is worth not worrying about too much if the machine is running fine :)

On google users say is potential kernel bug and missing some non critical driver for the running hardware 🤔
